src/System.Windows.Forms.Primitives/src/Windows/Win32/System/Com/IDispatch.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using Windows.Win32.System.Variant;
+using FDEX_PROP_FLAGS = Windows.Win32.System.Ole.FDEX_PROP_FLAGS;
```

Comment:
Is there a reason you don't include the namespace? If you want to make the methods a little cleaner you could also add a static using. Then you'd be able to just use `fdex...`.

```suggestion
using Windows.Win32.System.Ole;
using static Windows.Win32.System.Ole.FDEX_PROP_FLAGS;
```